pages/checkout/access.tsx

```diff
@@ -0,0 +1,107 @@
+import { Button, TextField, Typography } from "@equinor/eds-core-react"
+import type { NextPage } from "next"
+import { useRouter } from "next/router"
+import { ChangeEventHandler } from "react"
+import { useIntl } from "react-intl"
+import { useDispatch, useSelector } from "react-redux"
+import styled from "styled-components"
+
+import { CheckoutWizard } from "../../components/CheckoutWizard/CheckoutWizard"
+import { Container } from "../../components/Container"
+import { Dispatch, RootState } from "../../store"
+
+const Headline = styled(Typography).attrs(() => ({ variant: "h4" }))`
```

Comment:
Personally I find it easier to read if the variant is used as a prop on the component itself 🤷‍♀️